modules/base/rendering/renderablesphere.cpp

```diff
@@ -112,6 +112,22 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    constexpr openspace::properties::Property::PropertyInfo IsUsingColorMapInfo = {
+        "IsUsingColorMap",
+        "Using Color Map",
```

Comment:
```suggestion
    constexpr openspace::properties::Property::PropertyInfo UseColorMapInfo = {
        "UseColorMap",
        "Use Color Map",
```

? This is what it's called in the parameters below and it's important that these two strings map. Otherwise, it will not be correctly applied. 

If there is time, it would also be good to add an exmaple for renderablesphere that uses this feature, so that we have a way to test it and for users to see how it works